### ISSUE TITLE
Copying a data model

### ISSUE THREAD

**Issue by wvengen** [metabase/metabase#5163]
In several cases it can be useful to have multiple databases with the same data model. For example, a staging and production environment, or with [schema-based](https://influitive.io/our-multi-tenancy-journey-with-postgres-schemas-and-apartment-6ecda151a21f) multi-[tenancy](https://msdn.microsoft.com/en-us/library/aa479086.aspx#mlttntda_sdss).

Currently, one needs to setup the data model for each database or schema (which may be tedious). An improvement would be to allow copying the data model from one metabase database/schema to another.

A more advanced approach may be allow linking a Metabase database/schema to another one (but that may be more complex, e.g. when the staging and production environment would have slightly different database structures because they run a different version).

:arrow_down: Please click the :+1: reaction instead of leaving a `+1` or `update?` comment

**Comment by flamber** [comment]
This would probably depend on #2052/#6485 (perhaps even duplicate of one of those).
Related to #6496

**Comment by slavashaz** [comment]
@wvengen 
If anyone needs it, I've made my script and uploaded in my repository. https://github.com/slavashaz/Metabase-Dashboard-Cloner
you can clone your dashboard with automatic changes for another db or same db but different identical table
